pom.xml

```diff
@@ -253,225 +128,8 @@
                     </execution>
                 </executions>
             </plugin>
-
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>build-helper-maven-plugin</artifactId>
-                <version>3.4.0</version>
-            </plugin>
-
-            <plugin>
-                <artifactId>maven-jar-plugin</artifactId>
-                <version>3.3.0</version>
-                <configuration>
-                    <archive>
-                        <manifestEntries>
```

Comment:
@jandroav it would be nice to be able to keep this configuration

Reply:
they are here https://github.com/liquibase/liquibase-parent-pom/blob/cf273b14759d590a6f9b814aad86b700e49b8703/pom.xml#L375 but I guess we can change the default vendor for some other property which reflects Neo4j instead